app/src/Pages/DetailsPage/Details.tsx

```diff
@@ -19,12 +21,35 @@ function Details() {
   // Grabs the args/params passed in through useNavigate's URL
   const params = useParams(); 
 
+  const EMPTY_CARD_RATING_MSG : string = 'No card ratings'; 
+
   /*
   'card' state variable is optional because we are already using the 
   receivedStatePokemonCard from the ProductCard component. 
   */
   const [card, setCard] = useState<PokemonCard>(); 
   const [priceHistory, setPriceHistory ] = useState<PriceHistory[]>([]); 
+  const [cardRating, setCardRating] = useState<CardRating[]>([]); 
+
+  /**
+   * Could not make cardRatingAverage as a state variable because 
+   * it would need a default value. If you used a default value of 0, 
+   * then anytime you clicked to view a card's details, the card
+   * rating would display as 0. 
+   * NOTE: if there are no card ratings, will instead display 
+   *       message saying no card ratings have been given. 
+   */  
+  let cardRatingAverage : number | string; 
```

Comment:
If you wanted to, you could probably reduce these few lines down to ternary operator. Totally optional though and it might be best left as is because a ternary might get a little messy here having the reduce function.

Reply:
Good idea, that's much cleaner